packages/@tinacms/mdx/src/parse/mdx.ts

```diff
@@ -63,6 +63,15 @@ export function mdxJsxElement(
     const childField = template.fields.find(
       (field) => field.name === 'children'
     )
+    template.fields.forEach((field) => {
+      if (field.type === 'rich-text') {
+        const value = props[field.name]
+        if (typeof value !== 'string') {
+          // props[field.name] = toTinaMarkdown({ type: 'root', children: value }, field)
+        }
+        console.log(props[field.name])
+      }
+    })
```

Comment:
This doesn't seem to do anything (other than debug info)? I might be wrong - but maybe this code block shouldn't make it back to `main` ...

Reply:
Yep! Thanks for the callout, removed.